### PR TITLE
Submission for test

### PR DESCRIPTION
**Pull request by LB-JM**
Finished the test
- Setup
- Character
- Animator Controller
- Player Hands
- Camera
- Character Placement
- Test
- Assault Rifle Prop
- Timeline
- Human Hands Camera Animation
- Unity-chan Animation
- Animation Ease-In and Ease-Out
- Assault Rifle Prop Activation
- Bullet Hole Activation
- Bullet Impact VFX
- Muzzle Flash VFX
- Screen Glow VFX
- Sounds

Added
- Fixed unity-chan's shadow
- Added fade out animation on bullet holes

### CODE REVIEW

**Comment by LB-Ferdi-Fernandez** [comment]
@LB-JM You did not rebase this branch at all. If you are not sure what rebase means, please check the following resources:

[What is Git Rebase? [Intermediate Git Tutorial]](https://www.youtube.com/watch?v=_UZEXUrj-Ds)
[How to Use Git Rebase (freeCodeCamp)](https://www.freecodecamp.org/news/how-to-use-git-rebase/)
[git rebase | Atlassian Git Tutorial](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)

But basically, if you think of the commits as a tree, a rebase is like doing a cut-and-paste of your commits onto a different part of the tree.

----

I see that you edited the Cast Shadows property while in the RoomViewer scene.

Cast Shadow Override

This caused changes to the `RoomViewer.unity` file. You should have applied these overrides so that the changes are saved to the prefab instead:
Apply Cast Shadow Override

Basically, we only work with prefabs, we don't edit scenes in the work that we do.

----

You should have stopped the music the moment that the dj-disc-stop sound played, like this:

music stop

----

The idle animation of the camera (humanHandsCamera@shootingRange_Idle) should have been set to loop. I didn't mention that in the instructions, so I won't require that to be fixed.

----

You can just make more commits on `user/LB-JM/warehouse_rebased` and push them. This PR will update automatically.
